.changelog/21265.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```

Comment:
Sometime we label this as `security` and list the CVE fixed. Do we know if this affects `consul`?

Reply:
I don't think it affect consul, I tried runing govuln on Consul with older versions and it didn't show up.